htetl/extract/phones.py

```diff
@@ -50,11 +51,29 @@ def _norm(text):
 
     return '-'.join([norm_area_code, norm_first_three, norm_last_four])
 
+
+def ExtractPhones(df):
```

Comment:
PEP-8 nit. Should rename to `extractPhone`

Reply:
I changed this to `extract_phone` PEP-8 is snake case right?

Reply:
yes, you are correct. I apparently should go sleep (writing Python is my day job).